docs/howto/writing-migrations.txt

```diff
@@ -57,6 +57,7 @@ Then, to leverage this in your migrations, do the following::
 
     def forwards(apps, schema_editor):
         # Your migration code goes here
+        ...
```

Comment:
The three dots are added so that syntax highlight works properly if that's the line you refer to.